.travis.yml

```diff
@@ -18,6 +18,7 @@ branches:
   only:
     - master
     - /support\/.*/
+    - /feature\/.*/
```

Comment:
Should this still be here?

Reply:
I think we shouldn't expect devs to remember to manually add and remove each time.

Reply:
That sounds like a good idea, thanks.